### ISSUE TITLE
Add role to rotate certificates on a cluster

### ISSUE THREAD

**Issue by DanielKuzmenkoo** [rancherfederal/rke2-ansible#190]
There should be a role that will be able to rotate the certificate on rke2 cluster

**Comment by Daemonslayer2048** [comment]
I disagree with the idea of a role but chances are we could achieve this with a playbook. We will look into this, thank you for the feedback.
